src/guides/v2.3/comp-mgr/patching.md

```diff
@@ -75,14 +79,164 @@ index c8a6fef58d31..7d01c195791e 100644
+║                │                                                 │          │             │  - magento/module-cms           ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-3           │ Fixes issue 3                                   │ Optional │ Not applied │ Required patches:               ║
+║                │                                                 │          │             │  - MC-2                         ║
+║                │                                                 │          │             │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-3-V2        │ Updated fix for issue 3, replaces MC-3 patch    │ Optional │ N/A         │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╚════════════════╧═════════════════════════════════════════════════╧══════════╧═════════════╧═════════════════════════════════╝
+Magento 2 Enterprise Edition, version 2.3.5.0
+```
+
+The status table contains the following types of information:
+
+-  **Type**:
+
+   -  Optional—All individual patches from the MQP package and the [Magento Cloud Patches]({{ site.baseurl }}/cloud/project/project-patch.html) package are optional for {{ site.data.var.ee }} and {{ site.data.var.ce }} installations.
+   -  Deprecated—The individual patch is marked as deprecated by Magento and we recommend reverting it if you have applied it. After you revert a deprecated patch, it will no longer be displayed in the status table.
+
+-  **Status**:
+
+   -  Applied—The patch has been applied.
+   -  Not applied—The patch has not been applied.
+   -  N/A—The status of the patch cannot be defined due to conflicts.
+
+-  **Details**:
+
+   -  Affected components—The list of affected Magento modules.
+   -  Required patches—The list of required patches (dependencies).
+   -  Recommended replacement—The patch that is a recommended replacement for a deprecated patch.
+
+#### Apply individual patches
+
+{:.bs-callout-warning}
+We strongly recommend testing all patches in a staging or development environment before deploying to production. We also strongly recommend backing up your data before applying a patch. See [Back up and roll back the file system][].
+
+To apply a single patch, run the following command where `MAGETWO-XXXX` is the patch ID specified in the status table:
+
+```bash
+./vendor/bin/magento-patches apply MAGETWO-XXXX
+```
+
+You can also apply several patches at the same time by separating multiple patch IDs with spaces:
```

Comment:
```suggestion
Also, you can apply several patches at the same time by separating each additional patch ID with a space:
```

I make this suggestion for a couple of reasons. So you don't use IDs. And so it is interpreted as "one space"; so not loosely interpreted? Your call.